File: login_and_signup_web-master/lib/Model/CustPerson.dart
class CustPerson {
  int custPersonID;
  String custPersonFullName;
  String custPersonDesignation;
  String custPersonTelMob;
  String custPersonEmail;
  int customerID;
  String cPerUName;
  String cPerPwd;
  bool isActive;
  int lastUpdatedBy;
  String lastUpdatedDT;

  CustPerson({
    this.custPersonID,
    this.custPersonFullName,
    this.custPersonDesignation,
    this.custPersonTelMob,
    this.custPersonEmail,
    this.customerID,
    this.cPerUName,
    this.cPerPwd,
    this.isActive,
    this.lastUpdatedBy,
    this.lastUpdatedDT,
  });

  factory CustPerson.fromJson(Map<String, dynamic> json) {
    return CustPerson(
      custPersonID: json['custPersonId'],
      custPersonFullName: json['custPersonFullName'],
      custPersonDesignation: json['custPersonDesignation'],
      custPersonTelMob: json['custPersonTelMob'],
      custPersonEmail: json['custPersonEmail'],
      customerID: json['customerId'],
      cPerUName: json['cPerUName'],
      cPerPwd: json['cPerPwd'],
      isActive: json['isActive'],
      lastUpdatedBy: json['lastUpdatedBy'],
      lastUpdatedDT: json['lastUpdatedDt'],
    );
  }
}

File: AniDAAPI/AniDAAPI/Models/CustPerson.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace AniDAAPI.Models
{
    public partial class CustPerson
    {
        public int CustPersonId { get; set; }
        public String CustPersonFullName { get; set; }
        public String CustPersonDesignation { get; set; }
        public String CustPersonTelMob { get; set; }
        public String CustPersonEmail { get; set; }
        public int CustomerId { get; set; }
        public String CPerUName { get; set; }
        public String CPerPwd { get; set; }
        public bool isActive { get; set; }
        public int LastUpdatedBy { get; set; }
        public DateTime? LastUpdatedDt { get; set; }
    }
}